src/SingleReport/MultiTouch.cpp

```diff
@@ -0,0 +1,246 @@
+/*
+Copyright (c) 2021 ilufang
+See the readme for credit to other people.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#include "MultiTouch.h"
+
+// HID Report identifiers
+#define _HID_REPORTID_TOUCH 0x01
```

Comment:
This report ID is already used:
https://github.com/NicoHood/HID/blob/4bf6cd6167b1152a292bbc62793a85ab69070895/src/HID-Settings.h#L34-L36

What you are doing here is using a Multireport Device, not a single report it seems. Compare:
https://github.com/NicoHood/HID/blob/master/src/MultiReport/ImprovedMouse.cpp#L32
https://github.com/NicoHood/HID/blob/master/src/SingleReport/BootMouse.cpp#L31

So it would make sense to remove that line 37 from the descriptor. Please test, if it still works, I dont know that.

Reply:
See "Feature Report" issue below.

Reply:
It turns out that if I remove the report-ids of both input report and feature report, the HID device will still work. I have them deleted now.

But unfortunately I think we can't make a multi-report version due to the inability of responding to feature report requests under PluggableHID.